src/plugin/services/AdminPluginFileService.ts

```
@@ -0,0 +1,458 @@
+        return {
+          success: false,
+          pluginId,
+          path: '',
+          filesWritten: 0,
+          writtenFiles: [],
+          manifest: {} as AdminPluginManifest,
+          error: filesValidation.error,
+        };
+      }
+
+      const manifest = filesValidation.manifest!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)